src/modules/Registration/components/RegistrationForm.tsx

```diff
@@ -58,16 +64,15 @@ export function RegistrationForm() {
   const [registrationError, setRegistrationError, registrationErrorText] =
     useAuthError();
 
-  function signUp(data: SignUpCredentialsForm): void {
-    signUpUser(toSignUpCredentials(data)).then(
-      () => history.push(AUTH_PAGE),
-      () => setRegistrationError('sendError'),
-    );
-  }
+  // TODO: error handling should be moved to services' level
```

Comment:
I agree. I consider It should be moved to our `http` module in which we need to create an instance of error according to the code